workers/unity/Assets/Playground/Config/MetricConfig.cs

```diff
@@ -0,0 +1,12 @@
+namespace Playground
+{
+    public delegate float CalculateLoadDelegate(float fps);
+
+    public static class MetricConfig
+    {
+        public static CalculateLoadDelegate CalculateLoad;
+
+        public static int MaxFpsSamples = 50;
```

Comment:
Can probably just move these into the system. Nothing else consumes them and this doesn't really increase visibility.